src/classes/SftpSessionHandler.ts

```diff
@@ -517,63 +539,34 @@ export class SftpSessionHandler {
       this.sftpConnection.status(reqId, SFTP_STATUS_CODE.OK);
       return;
     }
-    const {
-      temporaryFile,
-    } = serverResource;
-    fs.stat(
-      temporaryFile.name,
-      (statError, stats) => {
-        if (statError) {
-          logger.verbose(
-            'Response: Status (FAILURE)',
-            {
-              reqId,
-              code: SFTP_STATUS_CODE.FAILURE,
-              path: temporaryFile.virtualPath,
-            },
-          );
-          this.sftpConnection.status(
-            reqId,
-            SFTP_STATUS_CODE.FAILURE,
-            'An error occurred when attempting to load the file statistics for the file associated with this handle.',
-          );
-          return;
-        }
-        const { size } = stats;
-        this.getCurrentPermanentFileSystem().createFile(
-          temporaryFile.virtualPath,
-          fs.createReadStream(temporaryFile.name),
-          size,
-        ).then(() => {
-          temporaryFile.removeCallback();
-          this.activeHandles.delete(handle.toString());
-          logger.verbose(
-            'Response: Status (OK)',
-            {
-              reqId,
-              code: SFTP_STATUS_CODE.OK,
-              path: temporaryFile.virtualPath,
-            },
-          );
-          this.sftpConnection.status(reqId, SFTP_STATUS_CODE.OK);
-        }).catch((err) => {
-          logger.debug(err);
-          logger.verbose(
-            'Response: Status (FAILURE)',
-            {
-              reqId,
-              code: SFTP_STATUS_CODE.FAILURE,
-              path: temporaryFile.virtualPath,
-            },
-          );
-          this.sftpConnection.status(
-            reqId,
-            SFTP_STATUS_CODE.FAILURE,
-            'An error occurred when attempting to register this file on Permanent.org.',
```

Comment:
I would expect to still see this error somewhere. I'm sure I'm missing it - help?

Reply:
Oh goodness -- see this is what happens when one tries to change too many things at once and doesn't have automated tests!  Really great catch.  Fixing this now.

Reply:
Status report: I've fixed this specific issue up; running some more robust manual tests though before asking for a re-review.  I'll ping later today when this is ready for you again!